app-server/app/src/main/scala/footprint/Footprint.scala

```diff
@@ -0,0 +1,67 @@
+package com.azavea.rf.footprint
+
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+import java.sql.Timestamp
+import java.util.UUID
+import spray.json._
+
+import geotrellis.vector.io._
+import geotrellis.vector.Geometry
+import geotrellis.slick.Projected
+
+import com.azavea.rf.utils.Database
+import com.azavea.rf.datamodel.latest.schema.tables._
+import com.azavea.rf.AkkaSystem
+
+case class FootprintGeojson(id: UUID, organizationId: UUID, createdAt: Timestamp, modifiedAt: Timestamp, polygon: JsValue) {
+  def this(footprint: FootprintsRow) = this(
+    footprint.id, footprint.organizationId, footprint.createdAt,
+    footprint.modifiedAt, footprint.polygon.geom.toGeoJson.toJson
+  )
+  def toFootprintsRow(): FootprintsRow = {
```

Comment:
Spacing

Also - what's the significance of `this` in the case of a case class. Below, references can be made directly to the attributes (e.g. instead of `this.id`, just `id`.